css/style.css

```diff
@@ -18,3 +19,14 @@ button {
   font-size: 18px;
   box-sizing: border-box;
 }
+
+h2 {
+  font-size: 28px;
+  font-weight: 400;
+}
+
+@media screen and (min-width: 1024px) {
+  h2 {
+    font-size: 70px;
+  }
+}
```

Comment:
@FranzGB you should move this to an individual file `elements.css`, also, even though you can use elements to style your site, is better to use a class for it, consider that an `h2` could be used with different styles on different contexts, therefore, by using a single class you can avoid having to override your classes